onnxscript/rewriter/ort_fusions/skip_normalization.py

```diff
@@ -93,10 +91,11 @@ def rewrite(
         skip,
         gamma,
         bias,
-        epsilon,
```

Comment:
Is epsilon an attribute with a default value in ORT's SimplifiedLayerNorm? If so, it should be specified and used in the `attributes.get_float("epsilon")` line down below. Otherwise, if it is not optional, there doesn't seem to be a reason for this line to change.